database/engine/rrdengineapi.h

```diff
@@ -119,8 +102,8 @@ typedef struct rrdengine_size_statistics {
 
     size_t single_point_pages;
 
-    usec_t first_t;
-    usec_t last_t;
+    time_t first_t;
+    time_t last_t;
```

Comment:
I will bring here what I already talked with another developer as suggestion, this is far from to be a blocker.

I understand what means the suffix `_t` for `first_t` and `last_t`, on the other hand we have a complete different meaning for the variable [type](https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html) that also has the suffix `_t`. My suggestion is to avoid `_t` for variable names and keep this pattern only for data type.

Of course compilers and computers don't care for this, they look only for offset, but probably it is more readable.

Reply:
Fixed them all I think. Now all `time_t` variables end with `_s`, not `_t`.